modules/con_enabling-config_management_plugins_in_an_argo_cd_cr.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-config_management_plugins_in_an_argo_cd_cr_{context}"]
+= Enabling Config Management Plugins in an Argo CD CR
+
+Argo CD provides support for Helm, Jsonnet, and Kustomize as built-in config management tools. To use a different config management tool, or to enable features not provided by the built-in config management tools, you can use the Config Management Plugin (CMP).
+
+In the {gitops-title} Operator, the Config Management Plugin (CMP) is specified as a sidecar container for the Argo CD repo server container. For more information, see Config Management Plugins.
+
+In the {gitops-title} Operator, you can configure the Config Management plugin as a sidecar container in the Argo CD custom resource (CR). When configuring the sidecar container, you can specify an off-the-shelf or custom-built container image.
```

Comment:
This should include the info that you can omit explicitly specifying the image.

```
When configuring the sidecar container, you can specify an off-the-shelf or custom-built container image, or you can omit specifying the image in which case the image used for the repo server will also be used for the plugin
```

Reply:
Fixed.